src/server/actions/webhooks.ts

```diff
@@ -0,0 +1,33 @@
+import dbConnect from '@/utils/db-connect';
+import CMError, { CMErrorType } from '@/utils/cmerror';
+import { BackgroundCheckWebhookPayload } from '@/types/dataModel/backgroundCheckWebhookPayload';
+import VolunteerSchema from '@/server/models/Volunteer';
+/**
+ * Get a specific volunteer and update background check
+ * @param payload // BackgroundCheckWebhookPayload
+ * @returns // volunteerschema or null
+ */
+export async function handleBackgroundCheckWebhook(
+  payload: BackgroundCheckWebhookPayload
+): Promise<void> {
+  let res;
+  try {
+    await dbConnect();
+    res = await VolunteerSchema.findOneAndUpdate(
+      {
+        email: payload?.data.employee_email,
+        backgroundCheck: { $exists: true },
+      },
+      {
+        $set: {
+          'backgroundCheck.status': payload.data.overall_status,
```

Comment:
make sure you don't just set the value here, it may come back as "PASSED" or "FAILED". instead, first make it so that only the first letter is capitalized like how we are expecting it. i.e. "Passed" "Failed"